src/library_path.js

```diff
@@ -64,9 +64,8 @@ addToLibrary({
       return root + dir;
     },
     basename: (path) => {
-      // EMSCRIPTEN return '/'' for '/', not an empty string
+      // EMSCRIPTEN return '/' for '/', not an empty string
       if (path === '/') return '/';
-      path = PATH.normalize(path);
```

Comment:
Can we land this as its own change, since it seems like it could effect other things.

Also, are there any other places where we could not use `basename`, maybe?  But we were not because of this normalization?  

It might even be worth a changelog entry, mentioning that this is how node anyway and so its consistent with `library_nodepath.js`

Reply:
Yeah it's also how `basename` works in coreutils.

Reply:
Opened #23180 with this change.